code/part-two/README.md

```diff
@@ -168,14 +168,25 @@ working with Javascript, start with these documents:
 
 ### App Development Lecture
 
-Start with this lecture covering application development on the Sawtooth
-platform. Videos of the lecture will be available soon, but the slides are
-available now in a variety of formats:
+In addition to reviewing the official Sawtooth documentation, watch the
+Cryptomoji video lecture, which covers all the basics of Sawtooth application
+development in detail. It is broken into five parts, the first of which is a
+overview of blockchains generally, which you may have already watched in part
+one. Each part is available as an MP4 file:
```

Comment:
Is `mp4` the same as `m4v`?

Reply:
M4V files are MP4s with a little extension for optionally adding DRM. VLC output the files in that format for whatever reason. I am going to change the extension, which should make some players happy, and should work fine since there isn't actually any DRM.